tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -3124,15 +3165,6 @@ TEST_P(ParameterizedSqueezeTest, ConvertSqueeze) {
           Status{error::INVALID_ARGUMENT,
                  "Axis value of -5 is out of bounds, must be in range [-4, 4), "
                  "at my_squeeze"}},
-      TestParamBase{
```

Comment:
I can't put the comment closer to the code, but can we change the error message above
"Squeeze is only implemented for explicit dims, at my_squeeze"
to
"Squeeze is only implemented for non-empty squeeze_dims, at my_squeeze"